examples/sample-layers/label-layer/label-layer.js

```diff
@@ -37,12 +41,14 @@ export default class LabelLayer extends CompositeLayer {
   }
 
   renderLayers() {
+    const {fontSize} = this.props;
+
     return new IconLayer(Object.assign({}, this.props, {
       id: 'label-icons',
       iconAtlas: this.state.texture,
       iconMapping: this.state.mapping,
       data: this.state.mapping,
-      sizeScale: 24,
+      sizeScale: fontSize,
```

Comment:
If we're still using the `fontSize`, should keep it in `defaultProps` with value: 1